ztp/argocd/resource-hooks/pre-sync.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: siteconfig-pre
+  namespace: ran-sites
+  generateName: pre-sync-
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/hook-delete-policy: BeforeHookCreation
+spec:
+  template:
+    spec:
+      containers:
+        - name: ztp-site-generator
+          image: quay.io/redhat_emp1/ztp-site-generator:latest 
```

Comment:
what's `redhat_emp1`?

Reply:
The Red Hat employee organization in Quay that we all can join to push / pool the same images, including private repos. See the Makefile for details